htdocs/ajax/search_p_dyn.php

```diff
@@ -21,6 +22,9 @@
     $lab_config_id = $_SESSION['lab_config_id'];
 }
 $search_settings = get_lab_config_settings_search($lab_config_id);
+$satellite_lab_id = get_satellite_lab_user_id($_SESSION['user_id']);
+$log->info("Got satellite lab id $satellite_lab_id for lab id $lab_config_id");
```

Comment:
This is fine for now, but it should eventually be removed once the satellite lab ID is passed to the function that scopes the queries!